main.py

```diff
@@ -5,12 +5,17 @@
 colorNum = 3
 graphPath = "graph1.gx"
 population = createInitialPopulation(graphPath, colorNum)
-
+bestGraph = None
+generations = 0
 # Careful! This might create infinite graphs
 while True:
+	generations+=1
```

Comment:
We could set a hyperparameter for maximum iterations and do while True or generations < max_iter

Reply:
Yeah, we probably should do it for graphs that never find a solution. Any ideas on the threshold or how we can calculate it?